build/JsonRpcSharp.nuspec

```diff
@@ -30,11 +30,21 @@
         </dependencies>
     </metadata>
     <files> 
-        <!--Net 4.6.1-->
-        <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\net461\JsonRpcSharp.*" target="lib\net461" />
-        <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\net461\JsonRpcSharp.*" target="lib\net461" />
+        <!--Net 4.6-->
+        <file src="..\src\JsonRpcSharp.Client\bin\Release\net451\JsonRpcSharp.*" target="lib\net46" />
+        <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\net461\JsonRpcSharp.*" target="lib\net46" />
+        <file src="..\src\JsonRpcSharp.RpcClient\bin\Release\net451\JsonRpcSharp.*" target="lib\net46" />
+        <file src="..\src\JsonRpcSharp.TcpClient\bin\Release\netstandard2.0\JsonRpcSharp.*" target="lib\net46" />
+        <file src="..\src\JsonRpcSharp.UnixIpcClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\net46" />
```

Comment:
netstandard1.3 lib for the net46 profile? Not sure, but I don't think this would work, mmm